csrc/ops/arith.cpp

```diff
@@ -1744,7 +1767,21 @@ WelfordResult Welford(
   }
 
   if (!reduction_axes.empty()) {
-    return WelfordRaw(squeezed, reduction_axes, init_avg, init_var, init_N);
+    DataType dtype = tv->getDataType().value();
+    if (dtype == DataType::ComplexFloat || dtype == DataType::ComplexDouble) {
```

Comment:
nit: use this: https://github.com/NVIDIA/Fuser/blob/9f9ba02d2d4aab6587ca76c3869baec4299c9459/csrc/type.h#L200

Reply:
done.